aiida/engine/processes/calcjobs/manager.py

```diff
@@ -189,63 +211,72 @@ def _has_job_state_changed(old, new):
         return old.job_state != new.job_state or old.job_substate != new.job_substate
 
     def _get_next_update_delay(self):
-        """
-        Calculate when we are next allowed to call the scheduler get jobs command
-        based on when we last called it, how long has elapsed and the minimum given
-        update interval.
+        """Calculate when we are next allowed to call the scheduler get jobs command.
 
-        :return: The delay (in seconds) for when it's safe to call the get jobs command
+        This delay will be calculated as the minimum safe interval that needs to be guaranteed, as defined by the
+        authentication info for this instance, minus the last time we updated the list.
+
+        :return: delay (in seconds) after which the minimum update interval has passed since last update
         :rtype: float
         """
-        if self._last_updated is None:
+        if self.last_updated is None:
+            self.logger.info('AuthInfo<{}>: next scheduler update will be in {} seconds'.format(self._authinfo.pk, 0))
```

Comment:
```suggestion
            self.logger.info('AuthInfo<{}>: next scheduler update allowed in {} seconds'.format(self._authinfo.pk, 0))
```

Reply:
Also, I'm wondering: how often can this be called per calculation?
Is there a risk of overflowing the log with this (even though it's the "INFO" level)?

Reply:
Not really, because if a new request comes in when a request is already active (from another calculation) it will be simply added and the delay is not requested. See the `_ensure_updating` method which will be a no-op when the `_update_handle` is already defined